client/c/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.0)
+project(yodaos_client_c)
+
+aux_source_directory(src SRC_LIST)
+include_directories("./include")
+
+set(HEADER_FILES
+    ./include/yodaos_sdk.h
+    ./include/yodaos_api_defines.h
+    ./include/yodaos_apis.h
+)
+
+if(EXISTS "$ENV{STAGING_DIR}")
+    include_directories("$ENV{STAGING_DIR}/usr/include/caps/")
```

Comment:
Prevent from using `include_directories` for dependencies directly. Try `add_dependencies`.